homework-submission/todo-app/src/todos/todos.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import Element from './element';
+import AddForm from './addform';
+import {inject, observer} from 'mobx-react';
+
+@inject('TodosStore')
+@observer
+
+class Todos extends Component {
+    render() {
+        return ( 
+            <div className="wrapper">
+                <div className="todo-container container-fluid col-md shadow p-2 mb-2 bg-white rounded">
+                    <h2>Todo List :</h2>
+                    <AddForm />
+                    {this.checkElementsMount()}
+                </div>
+            </div>
+         );
+    }
+
+    extractElements() {
+        const {TodosStore} = this.props;
+        return TodosStore.todos.map((el, index) =>
```

Comment:
Oh you have `index` is defined but not in use.
```suggestion
        return TodosStore.todos.map(el =>
```